src/damage_wrap.cxx

```diff
@@ -11,8 +11,10 @@ PYBIND11_DECLARE_HOLDER_TYPE(T, std::shared_ptr<T>)
 namespace neml {
 
 PYBIND11_MODULE(damage, m) {
+  py::module::import("neml.models");
```

Comment:
is this required?
it doesn't seem to be used anywhere

Reply:
It is definitely required on the python side.